courses/free-courses-en.md

```diff
@@ -567,6 +567,7 @@
 * [Learn SQL](https://popsql.com/learn-sql) - PopSQL
 * [Learn SQL](https://www.sololearn.com/learning/1060) - *registration required*
 * [Learn SQL: SQL Tutorial for Beginners](https://www.programiz.com/sql) - Programiz
+* [MySQL for Developers](https://planetscale.com/learn/courses/mysql-for-developers/introduction/course-introduction) - Aaron Francis (PlanetScale) _(email address required)_
```

Comment:
```suggestion
* [MySQL for Developers](https://planetscale.com/learn/courses/mysql-for-developers/introduction/course-introduction) - Aaron Francis (PlanetScale)
```
seems like you can access the content without registering

Reply:
It does gate it after a couple of videos so just wanted to mention it. I'm good to remove it though if it's unnecessary.

Reply:
+1 for removing this. Everything else looks good.

Once you have made the requested changes, please update the PR title to include “(changes made)” and a reviewer will take another look as soon as they are available. Thanks!

Reply:
Just updated! Thank you!